addons/website_crm_partner_assign/controllers/main.py

```diff
@@ -99,14 +98,14 @@ def portal_my_opportunities(self, page=1, date_begin=None, date_end=None, sortby
         domain = self.get_domain_my_opp(request.env.user)
 
         today = fields.Date.today()
-        this_week_end_date = fields.Date.to_string(fields.Date.from_string(today) + datetime.timedelta(days=7))
 
         searchbar_filters = {
             'all': {'label': _('Active'), 'domain': []},
+            'no_activities': {'label': _('No Activities'),
+                              'domain': ['|', ('activity_date_deadline', '=', False), ('activity_user_id', '!=', request.env.user.id)]},
```

Comment:
@nihp-odoo  Why you need `('activity_date_deadline', '=', False)` here?

Reply:
Hello @ibo-odoo ,
We are showing opportunities in 'No Activities' in 2 cases.
Case 1: When an opportunity created by a portal user has no activity scheduled at all.
Case 2: When an opportunity created by a portal user has an activity created by another user in the backend but does not include the portal user in it

Here, we are using `('activity_date_deadline', '=', False)` for the **Case 1**.

Thankss.. :)

Reply:
@nihp-odoo  Okay. But It isn't very clear for `no activity scheduled` case. So can you please try to check this case with any other activity specific domain? Give it a try and let me know.

Note: Condition ('activity_date_deadline', '=', False) is working because scheduling an activity without a deadline takes today's date as a default.

Reply:
Hello @ibo-odoo ,
Requested changes are done. Now we are using `activity_ids` instead of `activity_date_deadline`. Now I think it should be clear about `no activity scheduled`.
Thanks..